TodoApp/TodoApp.Api.Tests/Controllers/ItemsControllerTests.cs

```diff
@@ -62,65 +61,61 @@ public void TearDown()
         public async Task GetReturnsAllItems()
         {
             var actionResult = await _controller.GetAsync();
-
             var contentResult = await actionResult.ExecuteAsync(CancellationToken.None);
             contentResult.TryGetContentValue(out Item[] value);
 
             Assert.That(contentResult.Content, Is.Not.Null);
             Assert.That(contentResult.StatusCode, Is.EqualTo(HttpStatusCode.OK));
-            Assert.That(value, Is.EqualTo(_items).AsCollection.UsingItemEqualityComparer());
+            Assert.That(value, Is.EqualTo(_items).AsCollection.UsingItemEqualityComparer(),
+                "Collections are not equal");
         }
 
         [Test]
         public async Task GetReturnsItemWithSameId()
         {
             var actionResult = await _controller.GetAsync(Guid1);
-
             var contentResult = await actionResult.ExecuteAsync(CancellationToken.None);
             contentResult.TryGetContentValue(out Item item);
 
             Assert.That(contentResult.Content, Is.Not.Null);
             Assert.That(contentResult.StatusCode, Is.EqualTo(HttpStatusCode.OK));
-            Assert.That(item, Is.EqualTo(_items[0]).UsingItemEqualityComparer());
+            Assert.That(item, Is.EqualTo(_items[0]).UsingItemEqualityComparer(), "Items are not equal");
```

Comment:
I'm not saying assert message are wrong, quite contrary, however, it needs to bring value. You could say that "Returned item does not match posted item" or something like that, however, this won't bring the value just yet (since the implementation is dummy just now and would either be false now or be misleading in future). I vote for removing them for now.